app/controllers/users_controller.rb

```diff
@@ -5,6 +5,6 @@ def index
 
   def show
     @user = User.find(params[:id])
-    @posts = @user.recent_posts
+    @posts = Post.order(created_at: :desc).where(author_id: @user.id)
   end
 end
```

Comment:
- [ ] We don't implement SQL Queries inside the controller, we use the model to keep the SQL queries implemantation.

Reply:
Thank you for letting me know. I fixed it. 👍